src/vs/platform/terminal/node/terminalProcess.ts

```diff
@@ -121,6 +128,17 @@ export class TerminalProcess extends Disposable implements ITerminalChildProcess
 				}
 			}));
 		}
+		this.onProcessReady(() => {
+			if (platform.isWindows && this._ptyProcess?.pid) {
```

Comment:
Let's pull the `platform.isWindows` conditional out and wrap the above `if` as well to make it clear that these 2 chunks are related to windows.